projects/extension/src/background/index.ts

```diff
@@ -6,7 +6,7 @@ import { enqueueAsyncFn } from "./enqueueAsyncFn"
 import { trackChains } from "./trackChains"
 
 import * as environment from "../environment"
-import { PORTS } from "../shared"
+import { PORTS, wellKnownChainNames } from "../shared"
```

Comment:
Given that `wellKnownChainNames` is not shared by other contexts (content script or extension pages), let's remove it from `shared` and keep it local to this module